benchmark/dgram/bind-params.js

```diff
@@ -0,0 +1,42 @@
+﻿'use strict';
+
+const common = require('../common.js');
+const dgram = require('dgram');
+
+const numberOfOperations = 1e4;
+
+const configs = {
+  n: [numberOfOperations],
```

Comment:
OK. I've just linted with [no-magic-numbers](http://eslint.org/docs/rules/no-magic-numbers), but this is not Node.js mandatory rule. I will fix it.